pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[build-system]
+requires = ["setuptools", "numpy>=1.8", "cython"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "powerfit"
+version = "2.0.0"
```

Comment:
mind the version here, should bump it to v3.0.0

Reply:
Done